tests/test_senet.py

```diff
@@ -26,31 +26,33 @@
 
 device = "cuda" if torch.cuda.is_available() else "cpu"
 
-TEST_CASE_1 = [senet154(3, 2, 2)]
-TEST_CASE_2 = [se_resnet50(3, 2, 2)]
-TEST_CASE_3 = [se_resnet101(3, 2, 2)]
-TEST_CASE_4 = [se_resnet152(3, 2, 2)]
-TEST_CASE_5 = [se_resnext50_32x4d(3, 2, 2)]
-TEST_CASE_6 = [se_resnext101_32x4d(3, 2, 2)]
+net_args = {"spatial_dims": 3, "in_channels": 2, "num_classes": 2}
+TEST_CASE_1 = [senet154, net_args]
+TEST_CASE_2 = [se_resnet50, net_args]
+TEST_CASE_3 = [se_resnet101, net_args]
+TEST_CASE_4 = [se_resnet152, net_args]
+TEST_CASE_5 = [se_resnext50_32x4d, net_args]
+TEST_CASE_6 = [se_resnext101_32x4d, net_args]
```

Comment:
let's call this NET_ARGS then since it is a constant kind of?
and to avoid warnings below (cuz we have net_args below as well)